Adding desription for  built-in MDATP admin role (#3676)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md

After creating roles, you'll need to create a machine group and provide access to the machine group by assigning it to a role that you just created. 

>[!NOTE]

> >The Windows Defender ATP administrator (default) role has administrator permissions. The administrator permissions cannot be assigned to any other role. On groups assigned the Windows Defender ATP administrator role have access to all machine groups.

Reviewer: ```suggestion
>The Windows Defender ATP administrator (default) role has administrator permissions. The administrator permissions cannot be assigned to any other role. On groups assigned, the Windows Defender ATP administrator role has access to all machine groups.
```

> >The Windows Defender ATP administrator (default) role has administrator permissions. The administrator permissions cannot be assigned to any other role. On groups assigned the Windows Defender ATP administrator role have access to all machine groups.

Reviewer 2: ```suggestion
>The Windows Defender ATP administrator (default) role has administrator permissions. Administrator permissions cannot be assigned to any other role. Only those who are (?) assigned the Windows Defender ATP administrator role have access to all machine groups.
```

> >The Windows Defender ATP administrator (default) role has administrator permissions. Administrator permissions cannot be assigned to any other role. Only those who are assigned the Windows Defender ATP administrator role, has access to all machine groups.

Reviewer 2: ```suggestion
>The Windows Defender ATP administrator (default) role has administrator permissions. Administrator permissions cannot be assigned to any other role. Only those who are assigned the Windows Defender ATP administrator role have access to all machine groups.
```

2.	Click the drop-down button and select **Delete role**.

> ## Related topic

Reviewer: ```suggestion
## Related topics
```

> - [User basic permissions to access the portal](basic-permissions-windows-defender-advanced-threat-protection.md)

Reviewer 2: ```suggestion
- [User basic permissions to access the portal](basic-permissions.md)
```

> - [Create and manage machine groups](machine-groups-windows-defender-advanced-threat-protection.md)

Reviewer 2: ```suggestion
- [Create and manage machine groups](machine-groups.md)
```

Reviewer 3: line 85 doesn't look to have been updated
